cmd/sqlflowserver/e2e_common_cases.go

```diff
@@ -64,6 +64,7 @@ func caseShowDatabases(t *testing.T) {
 		"imdb":                    "",
 		"sqlflow_model_zoo":       "",
 		"sqlflow_public_models":   "",
+		"energy":                  "", // energy tutoral table
```

Comment:
`tutoral` typo?

Reply:
This one: https://github.com/sql-machine-learning/sqlflow/blob/develop/doc/tutorial/energe_lstmbasedtimeseries.md